commands/droplet_autoscale.go

```diff
@@ -0,0 +1,379 @@
+limitations under the License.
+*/
+
+package commands
+
+import (
+	"strconv"
+
+	"github.com/digitalocean/doctl"
+	"github.com/digitalocean/doctl/commands/displayers"
+	"github.com/digitalocean/godo"
+	"github.com/spf13/cobra"
+)
+
+func DropletAutoscale() *Command {
+	cmd := &Command{
+		Command: &cobra.Command{
+			Use:     "droplet-autoscale",
+			Aliases: []string{"das"},
+			Short:   "Display commands to manage droplet autoscale pools",
+			Long: `Use the subcommands of ` + "`" + `doctl compute droplet-autoscale` + "`" + ` to perform actions on Droplet Autoscale Pools.
+
+You can use droplet-autoscale to perform CRUD operations on a Droplet Autoscale Pools.`,
+		},
+	}
+	cmdDropletAutoscaleCreate := CmdBuilder(cmd, RunDropletAutoscaleCreate, "create", "Create new droplet autoscale pool", "", Writer, displayerType(&displayers.DropletAutoscale{}))
+
+	cmdDropletAutoscaleUpdate := CmdBuilder(cmd, RunDropletAutoscaleUpdate, "update", "Update an active droplet autoscale pool", "", Writer, displayerType(&displayers.DropletAutoscale{}))
+	AddStringFlag(cmdDropletAutoscaleUpdate, doctl.ArgAutoscaleID, "", "", "ID of the droplet autoscale pool", requiredOpt())
+
+	for _, c := range []*Command{
+		cmdDropletAutoscaleCreate,
+		cmdDropletAutoscaleUpdate,
+	} {
+		AddStringFlag(c, doctl.ArgAutoscaleName, "", "", "Name of the droplet autoscale pool", requiredOpt())
+		AddIntFlag(c, doctl.ArgAutoscaleMinInstances, "", 0, "Min number of members")
+		AddIntFlag(c, doctl.ArgAutoscaleMaxInstances, "", 0, "Max number of members")
+		AddStringFlag(c, doctl.ArgAutoscaleCpuTarget, "", "", "CPU target threshold")
+		AddStringFlag(c, doctl.ArgAutoscaleMemTarget, "", "", "Memory target threshold")
+		AddIntFlag(c, doctl.ArgAutoscaleCooldownMinutes, "", 0, "Cooldown duration")
+		AddIntFlag(c, doctl.ArgAutoscaleTargetInstances, "", 0, "Target number of members")
+
+		AddStringFlag(c, doctl.ArgSizeSlug, "", "", "Droplet size")
+		AddStringFlag(c, doctl.ArgRegionSlug, "", "", "Droplet region")
+		AddStringFlag(c, doctl.ArgImage, "", "", "Droplet image")
+		AddStringSliceFlag(c, doctl.ArgTag, "", []string{}, "Droplet tags")
+		AddStringSliceFlag(c, doctl.ArgSSHKeys, "", []string{}, "Droplet SSH keys")
+		AddStringFlag(c, doctl.ArgVPCUUID, "", "", "Droplet VPC UUID")
+		AddBoolFlag(c, doctl.ArgDropletAgent, "", true, "Enable droplet agent")
+		AddStringFlag(c, doctl.ArgProjectID, "", "", "Droplet project ID")
+		AddBoolFlag(c, doctl.ArgIPv6, "", true, "Enable droplet IPv6")
+		AddStringFlag(c, doctl.ArgUserData, "", "", "Droplet user data")
+	}
+
+	cmdDropletAutoscaleGet := CmdBuilder(cmd, RunDropletAutoscaleGet, "get", "Get an active droplet autoscale pool", "", Writer, displayerType(&displayers.DropletAutoscale{}))
+	AddStringFlag(cmdDropletAutoscaleGet, doctl.ArgAutoscaleID, "", "", "ID of the droplet autoscale pool", requiredOpt())
+	AddStringFlag(cmdDropletAutoscaleGet, doctl.ArgFormat, "", "", "Format of the output data")
```

Comment:
You shouldn't need to add the `format` flag here manually. When the displayer is implemented correctly, it will be added automatically. The help output will also have the column names populated automatically.